src/theory/bv/theory_bv_rewrite_rules_constant_evaluation.h

```diff
@@ -42,8 +42,9 @@ Node RewriteRule<EvalAnd>::apply(TNode node) {
   BitVector a = node[0].getConst<BitVector>();
   BitVector b = node[1].getConst<BitVector>();
   BitVector res = a & b;
-  
-  return utils::mkConst(res);
+
+  NodeManager* nm = node.getNodeManager();
+  return utils::mkConst(nm, res);
```

Comment:
Merge (apply to this occurrence and all the similar ones below)

Reply:
Done